### DISCUSSION TITLE
android-studio displays : The IDE seems to be launched with a script launcher ('bin/studio.sh')

### DISCUSSION THREAD

**Discussion by new-creation7** [pacstall/pacstall-programs#6730] [Q&A] [answered]
![androidStudioMessage](https://github.com/user-attachments/assets/aa32721b-ff8f-43fb-9395-d1918887d85a)

https://youtrack.jetbrains.com/articles/SUPPORT-A-56/How-to-handle-Switch-to-a-native-launcher-notification

https://github.com/pacstall/pacstall-programs/blob/daa376c9d6e74739f7ab370bbd8d7cb0cac753a4/packages/android-studio/android-studio.pacscript#L33

Will that be solved if in pacscript it's `ln -sf "/opt/${pkgname}/bin/studio" "${pkgdir}/usr/bin/${pkgname}"` (without `.sh`) ?

**Comment by new-creation7** [comment]
> sudo ln -sf "/opt/android-studio/bin/studio" "/usr/bin/android-studio"
[sudo] password for you: 
~> sudo update-desktop-database -q

running above commands makes that android studio's message disappear

**Reply by new-creation7** [reply]
Can I make a Pull Request, please?

**Reply by Elsie19** [reply]
Sure.

**Comment by Elsie19** [comment, marked as answer]
Solved in #6931.
